lib/node_modules/@stdlib/math/base/special/fast/alpha-max-plus-beta-min/README.md

```diff
@@ -143,20 +143,86 @@ for ( i = 0; i < 100; i++ ) {
 
 </section>
 
-<!-- /.references -->
+* * *
 
-<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+<section class="c">
 
-<section class="related">
+## C APIs
 
-* * *
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
 
-## See Also
+<section class="intro">
 
--   <span class="package-name">[`@stdlib/math/base/special/hypot`][@stdlib/math/base/special/hypot]</span><span class="delimiter">: </span><span class="description">compute the hypotenuse avoiding overflow and underflow.</span>
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/fast/alpha-max-plus-beta-min.h"
+```
+
+#### double stdlib_base_fast_ampbm( const double x, const double y ) 
+
+Computes the hypotenuse using the alpha max plus beta min algorithm.
+
+```c
+double h = stdlib_base_fast_ampbm( -5.0, 12.0 ); // returns ~13.5
```

Comment:
This file needs to be rechecked, I suggest you to refer https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/fast/abs/README.md